tracekit.js

```diff
@@ -1071,54 +1071,58 @@ TraceKit.computeStackTrace = (function computeStackTraceWrapper() {
-            if (curr === computeStackTrace || curr === TraceKit.report) {
-                continue;
-            }
+                item = {
+                    'url': null,
+                    'func': UNKNOWN_FUNCTION,
+                    'args': [],
+                    'line': null,
+                    'column': null
+                };
 
-            item = {
-                'url': null,
-                'func': UNKNOWN_FUNCTION,
-                'args': [],
-                'line': null,
-                'column': null
-            };
+                if (curr.name) {
+                    item.func = curr.name;
+                } else if ((parts = functionName.exec(curr.toString()))) {
+                    item.func = parts[1];
+                }
 
-            if (curr.name) {
-                item.func = curr.name;
-            } else if ((parts = functionName.exec(curr.toString()))) {
-                item.func = parts[1];
-            }
+                if (typeof item.func === 'undefined') {
+                    try {
+                        item.func = parts.input.substring(0, parts.input.indexOf('{'));
+                    } catch (e) { }
+                }
 
-            if (typeof item.func === 'undefined') {
-              try {
-                item.func = parts.input.substring(0, parts.input.indexOf('{'));
-              } catch (e) { }
-            }
+                if ((source = findSourceByFunctionBody(curr))) {
+                    item.url = source.url;
+                    item.line = source.line;
 
-            if ((source = findSourceByFunctionBody(curr))) {
-                item.url = source.url;
-                item.line = source.line;
+                    if (item.func === UNKNOWN_FUNCTION) {
+                        item.func = guessFunctionName(item.url, item.line);
+                    }
 
-                if (item.func === UNKNOWN_FUNCTION) {
-                    item.func = guessFunctionName(item.url, item.line);
+                    var reference = / '([^']+)' /.exec(ex.message || ex.description);
+                    if (reference) {
+                        item.column = findSourceInLine(reference[1], source.url, source.line);
+                    }
                 }
 
-                var reference = / '([^']+)' /.exec(ex.message || ex.description);
-                if (reference) {
-                    item.column = findSourceInLine(reference[1], source.url, source.line);
+                if (funcs['' + curr]) {
+                    recursion = true;
+                } else {
+                    funcs['' + curr] = true;
                 }
-            }
 
-            if (funcs['' + curr]) {
-                recursion = true;
-            }else{
-                funcs['' + curr] = true;
+                stack.push(item);
             }
-
-            stack.push(item);
-        }
+        } 
+        catch (error) {
+            console.log(error)
```

Comment:
Sure, would re throwing the error be the preferred way to log the error? The concern with re throwing it is we are using this with Raygun4js and the actual error thrown causes another function not to return. Concern is this error failing silently.